swap/log.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019 The Swarm Authors
+// This file is part of the Swarm library.
+//
+// The Swarm library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Swarm library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Swarm library. If not, see <http://www.gnu.org/licenses/>.
+
+package swap
+
+import (
+	log "github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/ethersphere/swarm/network"
+)
+
+const (
+	// UndefinedAction is the default actions filter for swap logs
+	UndefinedAction string = "undefined"
+	// InitAction used when starting swap
+	InitAction string = "init"
+	// StopAction used when stopping swap
+	StopAction string = "stop"
+	// UpdateBalanceAction used when updating balances
+	UpdateBalanceAction string = "update_balance"
+	// SendChequeAction used for cheque actions
+	SendChequeAction string = "send_cheque"
+	// HandleChequeAction used for cheque actions
+	HandleChequeAction string = "handle_cheque"
+	// CashChequeAction used for cheque actions
```

Comment:
Could you be more explicit in the comments? 3 the same comments for 3 different constants.

Reply:
>     * instead of calling it `logger`, can we consider `log`? Saves quite a lot of typing :) Up to you!

i think _logger_ and _log_ can mean different things.

i only ask that we are consistent within this PR and also with the rest of the swarm code.

Reply:
fixed